errors.ts

```diff
@@ -64,26 +64,38 @@ function resolveCallStack(error: Error): string {
 	return outputMessage;
 }
 
+let attachUncaughtExceptionHandler: any;
 export function installUncaughtExceptionListener(): void {
-	process.on("uncaughtException", (err: Error) => {
-		let callstack = err.stack;
-		if (callstack) {
-			callstack = resolveCallStack(err);
-		}
-		console.error(callstack || err.toString());
+	if(!attachUncaughtExceptionHandler) {
+		attachUncaughtExceptionHandler = _.once(() => {
+			process.on("uncaughtException", (err: Error) => {
+				let callstack = err.stack;
+				if (callstack) {
+					try {
+						callstack = resolveCallStack(err);
+					} catch(err) {
+						console.error("Error while resolving callStack:", err);
+					}
+				}
+				console.error(callstack || err.toString());
+
+				if(!$injector.resolve("staticConfig").disableAnalytics) {
+					try {
+						let analyticsService = $injector.resolve("analyticsService");
+						analyticsService.trackException(err, callstack);
+					} catch (e) {
+						// Do not replace with logger due to cyclic dependency
+						console.error("Error while reporting exception: " + e);
+					}
+				}
 
-		if(!$injector.resolve("staticConfig").disableAnalytics) {
-			try {
-				let analyticsService = $injector.resolve("analyticsService");
-				analyticsService.trackException(err, callstack);
-			} catch (e) {
-				// Do not replace with logger due to cyclic dependency
-				console.error("Error while reporting exception: " + e);
-			}
-		}
+				// TODO: Find what to do here for CLIs
+				// process.exit(ErrorCodes.UNKNOWN);
```

Comment:
Pfff, I already know what to do, but I forgot to add it in this PR.
Thanks for pointing it :) I'll fix it